h1st/model/oracle/student_modelers.py

```diff
@@ -0,0 +1,47 @@
+from typing import Any, Dict
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.linear_model import LogisticRegression
+from sklearn.preprocessing import StandardScaler
+
+from h1st.model.ml_modeler import MLModeler
+from h1st.model.oracle.student_models import RandomForestModel, LogisticRegressionModel
+
+
+class RandomForestModeler(MLModeler):
```

Comment:
Why are `StudentModelers` needed? Aren’t they just regular models? The only thing special is where they fit within an `Oracle`, which is Oracle code. Not some special `StudentModeler` code.